### PR TITLE
Preact logger now backwards compatible

### PR DESCRIPTION
**Pull request by scarow**
User can log with _lnq or _preactq

### CODE REVIEW

**Comment by msaldi** [comment]
Hi @harrietgrace - we have a few joint customers who are looking for this update. Any chance we could get this merged in the near future?  Thanks!

File: lib/preact/index.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ var alias = require('alias');
 
 var Preact = module.exports = integration('Preact')
   .assumesPageview()
-  .global('_lnq')
+  .global('_preactq')
```
Comment (ndhoule) [inline lib/preact/index.js]: Just to make sure I understand—loading the Preact library now declares two globals, `_lnq` and `_preactq`, correct? If so, we should declare both globals by adding `.global('_lnq')` here.

File: lib/preact/test.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ describe('Preact', function(){
   it('should have the right settings', function(){
     analytics.compare(Preact, integration('Preact')
       .assumesPageview()
-      .global('_lnq')
+      .global('_preactq')
```
Comment (ndhoule) [inline lib/preact/test.js]: Here too—same change as previous comment.

**Comment by ndhoule** [comment]
Thanks for this! @h0ke @scarow, looks good to me other than the comment above. Can you make that change and I'll get this merged?

**Comment by scarow** [comment]
Hey @ndhoule, we're getting some failing specs for the LiveChat integration; not sure why. Can you advise? Thanks!

cc: @h0ke

**Comment by ndhoule** [comment]
@scarow: Not sure; seems unrelated to the changes you've made.

@yields was the last person touching LiveChat—Amir, any idea what's going on here?

**Comment by yields** [comment]
ah yeah, the livechat trial expired but i updated it recently, if you rebase on top of master the tests should pass.

**Comment by scarow** [comment]
Hey y'all -- specs are passing again after rebase, so I believe we are good
to go. @ndhoule please let me know if there are any more changes we need to
make!

On Sat, Jan 10, 2015 at 2:44 AM, Amir Abu Shareb <notifications@github.com>
wrote:

> ah yeah, the livechat trial expired but i updated it recently, if you
> rebase on top of master the tests should pass.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/segmentio/analytics.js-integrations/pull/488#issuecomment-69451615>
> .
>



-- 
*Samantha Carow*
Web Developer

p. (908) 370-4534
g. https://github.com/scarow

**Comment by ndhoule** [comment]
Hey @scarow, looks like the rebase wasn't clean, so this PR contains a bunch of already-merged commits. I tried to rebase locally (`git fetch upstream && git rebase upstream/master`) but got merge conflicts. Mind taking a look?

When you're done with that, can you also squash this into a single commit?

Thanks again!

**Comment by scarow** [comment]
@ndhoule hey we fixed the rebase & squashed into a single commit, but now we are getting spec failures on Mixpanel's project.

Please advise!

**Comment by ndhoule** [comment]
We're investigating, but I'm getting the same thing—seems totally unrelated to your changeset.

Thanks again for contributing this! Scheduling this for a release later this afternoon.

**Pull request merged**